packages/components/src/components/InteractiveOverlay/InteractiveOverlay.tsx

```diff
@@ -88,7 +118,9 @@ const Wrapper = styled(
     className: styledClassName,
   }: WrapperProps) => {
     if (React.Children.count(children) > 1) {
-      throw new Error('The Wrapper component can only contain one child.');
+      throw new Error(
+        'The StyledOverlay component can only contain one child.'
```

Comment:
What is StyledOverlay from the perspective of the user? (design system consumer). Should we reference the InteractiveOverlay component instead?

Reply:
Sharp, will update this!